charts/telepresence/README.md

```diff
@@ -24,77 +24,81 @@ Notable chart changes are listed in the [CHANGELOG](./CHANGELOG.md)
-| hooks.busybox.image                            | The name of the image to use for busybox.                                                                                   | `busybox`                                                                   |
-| hooks.busybox.tag                              | Override the version of busybox to be installed.                                                                            | `latest`                                                                    |
-| hooks.busybox.imagePullSecrets                 | The `Secret` storing any credentials needed to access the image in a private registry.                                      | `[]`                                                                        |
-| hooks.curl.registry                            | The repository to download the image from.                                                                                  | `docker.io`                                                                 |
-| hooks.curl.image                               | The name of the image to use for curl.                                                                                      | `curlimages/curl`                                                           |
-| hooks.curl.tag                                 | Override the version of busybox to be installed.                                                                            | `latest`                                                                    |
-| hooks.curl.imagePullSecrets                    | The `Secret` storing any credentials needed to access the image in a private registry.                                      | `[]`                                                                        |
-| client.connectionTTL                           | The time that the traffic-manager will retain a client connection without any sign of life from the workstation             | `24h`                                                                       |
-| client.routing.alsoProxySubnets                | The virtual network interface of connected clients will also proxy these subnets                                            | `[]`                                                                        |
-| client.routing.neverProxySubnets               | The virtual network interface of connected clients never proxy these subnets                                                | `[]`                                                                        |
-| client.routing.allowConflictingSubnets         | Allow the specified subnets to be routed even if they conflict with other routes on the local machine.                      | `[]`                                                                        |
```

Comment:
This line seems to have been inadvertently removed (it was added just recently, so perhaps a merge mistake?)

Reply:
Yes, my mistake. I'll fix this during the day.